src/test/integrasjonstester/kotlin/no/nav/familie/ks/sak/barnehagelister/BarnehagebarnServiceHentBarnIntegrasjonsTest.kt

```diff
@@ -161,6 +161,30 @@ class BarnehagebarnServiceHentBarnIntegrasjonsTest(
             )
     }
 
+    @Test
+    @Sql(
+        scripts = ["/barnehagelister/barnehagebarn-med-fulltid-barnehageplass.sql"],
+        executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD,
+    )
+    fun `hentBarnehageBarn oppgir ikke avvik sålenge barnet har fulltidsplass selv hvis det er forskjellig antall timer`() {
+        // Arrange
+        val barnehagebarnRequestParams1 =
```

Comment:
Trenger kanskje ikke "1" bak variablenavnet

```suggestion
        val barnehagebarnRequestParams =
```